### ISSUE TITLE
I'm getting - string value was longer than the available storage size of the specified column

### ISSUE THREAD

**Issue by adamrossnelson** [Roche/pyreadstat#92]
Please read the README, particularly the known limitations section!

Describe the issue
I'm reproducing the error [reported here](https://github.com/Roche/pyreadstat/issues/55).

```Python
>>> pyreadstat.write_sav(df, 'auto_spss_file.sav')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pyreadstat/pyreadstat.pyx", line 712, in pyreadstat.pyreadstat.write_sav
  File "pyreadstat/_readstat_writer.pyx", line 754, in pyreadstat._readstat_writer.run_write
  File "pyreadstat/_readstat_writer.pyx", line 742, in pyreadstat._readstat_writer.run_write
  File "pyreadstat/_readstat_parser.pyx", line 768, in pyreadstat._readstat_parser.check_exit_status
pyreadstat._readstat_parser.ReadstatError: A provided string value was longer than the available storage size of the specified column
```

To Reproduce
```
>>> import pandas as pd
>>> file = 'http://www.stata-press.com/data/r15/auto2.dta'
>>> df = pd.read_stata(file)
>>> pyreadstat.write_sav(df, 'auto_spss_file.sav')
```

File example
http://www.stata-press.com/data/r15/auto2.dta

Expected behavior
I would expect the file to write.

Setup Information:
How did you install pyreadstat?

pip

Platform (windows, macOS, linux, 32 or 64 bit)

macOS

Python Version

Python 3.8.3 (default, Jul  2 2020, 11:26:31)

Python Distribution (System, plain python, Anaconda)

Anaconda

**Comment by ofajardo** [comment]
thanks a lot for the report and the fully reproducible example. 

The problem is solved on the branch "dev_redst_115_rc0" where I am preparing the next release. You can compile from there if you wish, or you can wait for the next release which should come soon.

In the meantime, if you wish you can also read the stata file with pyreadstat instead of pandas, as the resulting dataframe is writable to spss:

```
import pyreadstat

df, meta = pyreadstat.read_dta("auto2.dta", apply_value_formats=True)
pyreadstat.write_sav(df, "auto_spss.sav")
```

**Comment by ofajardo** [comment]
issue solved in version 1.0.6 (just released)
